docs/cloud/cloud-troubleshooting/issue-creating-database.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Creating database issues
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 5
+---
+
+# Creating database issues.
```

Comment:
A question mark "?" is the only punctuation you can add to the end of a heading. And only when there's a question (e.g., "How do I <do something>?"